cli/job.go

```
@@ -69,7 +71,30 @@ func runJobCmd() *cobra.Command {
 			logger := log.NewCtxLogger(config.LogLevel, []string{config.AuditLogTraceIDHeaderKey})
 			crypto := crypto.NewAES(config.EncryptionSecretKeyKey)
 			validator := validator.New()
-			notifier, err := notifiers.NewClient(&config.Notifier, logger)
+			var notifierMap map[string]interface{}
+			errr := json.Unmarshal([]byte(config.Notifiers), &notifierMap)
+			if errr != nil {
+				fmt.Println(errr)
```

let's return error instead of just logging